tests/config/mod.rs

```diff
@@ -124,7 +124,7 @@ fn test_serde_custom_tikv_config() {
         raft_store_max_leader_lease: ReadableDuration::secs(12),
         right_derive_when_split: false,
         allow_remove_leader: true,
-        use_delete_range: true,
+        use_delete_range: false,
```

Comment:
true is ok

Reply:
In config test, we prefer don't use default value.